src/util/styleByDataItem.js

```diff
@@ -5,24 +5,29 @@ import {
     EVENT_RADIUS,
     CLASSIFICATION_PREDEFINED,
 } from '../constants/layers.js'
-import { numberValueTypes } from '../constants/valueTypes.js'
+import { numberValueTypes, booleanValueTypes } from '../constants/valueTypes.js'
 import { cssColor } from '../util/colors.js'
 import { OPTION_SET_QUERY, LEGEND_SET_QUERY } from '../util/requests.js'
 import { getLegendItemForValue } from './classify.js'
 import { getAutomaticLegendItems, getPredefinedLegendItems } from './legend.js'
 
+// Returns true if value is not undefined or null;
```

Comment:
No need for the comment - the code is very clear (and also the comment doesn't include the last possibility `value !== ''`